adm_program/system/classes/TableAccess.php

```diff
@@ -188,6 +191,151 @@ public function countAllRecords(): int
         return (int) $countStatement->fetchColumn();
     }
 
+    /**
+     * Return a human-readable representation of this record.
+     * If a column [prefix]_name exists, it is returned, otherwise the id.
+     * This method can be overridden in child classes for custom behavior.
+     * 
+     * @return string The readable representation of the record (can also be a translatable identifier)
+     */
+    public function readableName(): string
+    {
+        if (array_key_exists($this->columnPrefix.'_name', $this->dbColumns)) {
+            return $this->dbColumns[$this->columnPrefix.'_name'];
+        } elseif (array_key_exists($this->columnPrefix.'_headline', $this->dbColumns)) {
+            return $this->dbColumns[$this->columnPrefix.'_headline'];
+        } else {
+            return $this->dbColumns[$this->keyColumnName];
+        }
+    }
+
+    /**
+     * Return a human-readable representation of the given database field/column.
+     * By default, the column name is returned unmodified. Subclasses can override this method.
+     * @param string $field The database column
+     * @return string The readable representation of the DB column (can also be a translatable identifier)
+     */
+    public function getFieldTitle(string $field): string
+    {
+        return $field;
+    }
+
+    /**
+     * Retrieve the list of database fields that are ignored for the changelog.
+     * Some tables contain columns _usr_id_create, timestamp_create, etc. We do not want
+     * to log changes to these columns. Subclasses can also add further fields 
+     * (e.g. the users table stores and auto-increments the login count, which 
+     * we do not want to log)
+     * 
+     * @return true Returns the list of database columns to be ignored for logging.
+     */
+    public function logIgnoredColumns(): array
```

Comment:
@In my opinion `getIgnoredLogColumns` is a better name, because it tells better what the function does.

Reply:
changed.